rtloader/common/builtins/datadog_agent.c

```diff
@@ -116,6 +119,12 @@ void _set_tracemalloc_enabled_cb(cb_tracemalloc_enabled_t cb)
     cb_tracemalloc_enabled = cb;
 }
 
+void _set_obfuscate_sql_cb(cb_obfuscate_sql_t cb)
+{
+    cb_obfuscate_sql = cb;
+}
+
+
```

Comment:
A nit but could you remove the extra new line here.

Reply:
Removed